enhancements/installer/component-selection.md

```diff
@@ -103,44 +103,65 @@ on installer input. For example the console related resources could be annotated
-such as `capability.openshift.io/defaultDisable`.  If we do so, we will need to define how that choice
-interacts with the cluster scoped "default capabilities on/off" configuration setting.  For now, this
-EP takes the position that individual components that exist in the payload cannot choose to be
-default-disabled.
+capabilities they can include/exclude when configuring the installation.
+
+Note: Resources are enabled unless explicitly excluded by configuration.  If a component is defined
+as a capability than it can be included or excluded depending upon the InstallConfig.capabilities API.
+If the default is set to exclude i.e. `inclusionDefault: Exclude` then all capabilities will be excluded
+unless specifically included. Similarly if the default is include then all capabilities will be
+included in the install config unless excluded explicitly.  If `inclusionDefault` is not specified,
+it is treated as (defaults to) `Include` which ensures backwards compatibility with the existing behavior
+of always installing all components on the cluster.  In the future we may allow resources to be excluded
+unless explicitly included, this can be done by introducing a new annotation such as 
+`capability.openshift.io/defaultDisable`.  If we do so, we will need to define how that choice interacts
+with the cluster scoped "default capabilities on/off" configuration setting.  For now, this EP takes the
+position that individual components that exist in the payload cannot choose to be default-disabled.
 
 2) Defining an [install config api](https://github.com/openshift/installer/blob/790048166067273d34f76bea4220fa395b1cce1b/pkg/types/installconfig.go#L70) field whereby the user can opt out of specific capabilities.
 
+Example 1: include all capabilities by default, exclude some specific capabilities
 ```yaml
 InstallConfig.capabilities
-  inclusionDefault: [Include|Exclude]
+  inclusionDefault: Include
   exclude:
   - console
   - samples
+```
+
+Example 2: exclude all capabilities by default, include some specific capabilities.
+```yaml
+InstallConfig.capabilities
+  inclusionDefault: Exclude
   include:
   - registry
 ```
 
-Which resources ultimately get installed for a given cluster would be the set of resources encompassed
-by the CLUSTER_PROFILE(if any), minus any resources explicitly excluded by the excluded capabilities configuration,
-plus any explicitly included capabilities.  Any resource that is not excluded by the CLUSTER_PROFILE, is not
-included via the Include stanza, and is not Excluded via the Exclude stanza, will be filtered or not based on
-the `inclusionDefault` value (Include or Exclude).  The `inclusionDefault` field itself will default to `Include`.
+
+To determine which resources ultimately get installed for a given cluster:
+
+1) start with all the resources in the payload
+2) if a CLUSTER_PROFILE is configured, remove any resources that don't match the configured CLUSTER_PROFILE
```

Comment:
nit: [the CVO has a default profile][1], so there is no need for the `if a CLUSTER_PROFILE is configured` conditional.  And if you end with `the configured [cluster profile](/enhancements/update/cluster-profiles.md)` you don't have to worry about details like implicitly or explicitly declared profiles.

[1]: https://github.com/openshift/cluster-version-operator/blame/3db74ebb012838ffdd776ca65210e4d6d66b7472/pkg/payload/payload.go#L99

Reply:
updated